examples/http-server/default.conf

```diff
@@ -1,7 +1,10 @@
-SecDebugLogLevel 9
-SecDebugLog /dev/stdout
+SecRuleEngine DetectionOnly
+		SecAuditEngine On
+		SecAuditLogFormat json
+		SecAuditLogType serial
+		SecAuditLogParts ABCHZ
+		SecAuditLogRelevantStatus ".*"
+		# auditlog should not contain messages section without H flag included, but should contain Rule below
+		SecRule ARGS "@unconditionalMatch" "id:1,phase:1,log,auditlog,msg:'nolog message'"
 
-SecRule ARGS:id "@eq 0" "id:1, phase:1,deny, status:403,msg:'Invalid id',log,auditlog"
```

Comment:
Changes to `default.conf` are not meant to be merged, but here just for local testing. Am I right?

Reply:
🙌

Reply:
Sorry, I've done it by mistake